R/gen_functions.R

```diff
@@ -84,11 +84,11 @@ gen_plates <- function(n_plates, map_x, map_y = map_x,
 #' @export
 #' 
 gen_world <- function(n_plates, spread, weight = 1, noise = 50,
-                      map_x, map_y = map_x,
+                      map_x, map_y = map_x, seed = sample(0:(2^31 - 1), 1),
                       gravity_range = c(0.2, 1),
                       height_range = c(-10, 10),
                       dist_method = c("square", "manhattan")) {
-
+  set.seed(seed)
```

Comment:
I have heard terrible tales of functions setting seeds inside themselves. But looking at this I can't see that it would be a problem. It is certainly not a problem for us right now; just something we might need to change in the future if we end up wanting to put this on CRAN.